ios/NativeBridge.swift

```diff
@@ -213,10 +217,18 @@ extension CredentialsManagerError {
         switch self {
             case .noCredentials: code = "NO_CREDENTIALS"
             case .noRefreshToken: code = "NO_REFRESH_TOKEN"
-            case .renewFailed: code = "RENEW_FAILED"
+            case .renewFailed: if let cause = self.cause as? AuthenticationError {
+                code = cause.code
+            } else {
+                code = "RENEW_FAILED.UNKNOWN"
+            }
             case .storeFailed: code = "STORE_FAILED"
             case .biometricsFailed: code = "BIOMETRICS_FAILED"
-            case .revokeFailed: code = "REVOKE_FAILED"
+            case .revokeFailed: if let cause = self.cause as? AuthenticationError {
+                code = cause.code
+            } else {
+                code = "REVOKE_FAILED.UNKNOWN"
```

Comment:
Changing this code from "REVOKE_FAILED" to "REVOKE_FAILED.UNKNOWN" would be a breaking change, I think. This is not the case for the "OTHER" case a few lines of code above, because we're keeping this "OTHER" code intact, so the whole change amounts to adding new error codes (the ones from the server).
Whereas here the old code ("REVOKE_FAILED") is being removed.